ingestion/tests/unit/test_fqn.py

```diff
@@ -90,8 +91,10 @@ def test_build_table(self):
         """
         Validate Table FQN building
         """
+        mocked_metadata = MagicMock()
+        mocked_metadata.es_search_from_fqn.return_value = None
         table_fqn = fqn.build(
-            ...,  # metadata client not needed with all params
+            metadata=mocked_metadata,
```

Comment:
hehe I was a bit lazy when writing this test xD

Reply:
No no, it was fine before but now we need to call ES first ;)